http/v2.md

```diff
@@ -431,7 +431,7 @@ Returns a HTTP 201 response with the following payload:
 
 Error response will be sent with a non-2xx HTTP status code. See the [Errors](#errors) section for more information.
 
-### GET /v2/fee_recipients
+### POST /v2/fee_recipients
```

Comment:
I think this should be `/v2/order_config` endpoint not `fee_recipients`?

Reply:
😱